Update advanced-configuration.md

Wonderingg opened the pull request:
### Type
- [x] Rewrite already written documentation.
- [ ] Add/remove new pages.

### Questions
- [ ] Did you preview on your local machine beforehand?
- [x] Did you follow our [commit handbook](https://github.com/Atlas-OS/docs/blob/master/.github/CONTRIBUTING.md#commit-message)?

### Describe your pull request
fix: 🐛 Boot Configuration
fix: 🐛 NVIDIA Display Container

JayXTQ reviewed (changes requested):
If you wish for this to be pushed, please provide more information.

Wonderingg commented:
> If you wish for this to be pushed, please provide more information.

More information about the files i changed or more information in the docs that i changed?

JayXTQ commented:
More information about the topics you wrote for. Check the configuration based pages for more examples of this
